src/hooks/data/useData/index.tsx

```diff
@@ -134,6 +134,12 @@ export type TSVResponse = Array<string>[];
 
 export type FASTAResponse = Array<string>[];
 
+export type V2Obj = {
+  study_accession: string;
+  accession: string;
+  downloads: Array<string>;
+};
+
```

Comment:
I think these should probably all be optional params on the type? If this is intended to be a generic type for V2 endpoints?